src/hotspot/share/gc/g1/g1_globals.hpp

```diff
@@ -304,7 +304,7 @@
           "disables this check.")                                           \
           range(0.0, (double)max_uintx)                                     \
                                                                             \
-  product(bool, G1AllowProactiveGC, true, DIAGNOSTIC,                       \
+  product(bool, G1AllowPreventiveGC, true, DIAGNOSTIC,                       \
           "Allows collections to be triggered proactively based on the      \
```

Comment:
Intentionally left "proactive" here (and in a comment in g1Policy.hpp) intentionally since it does describe how the feature works.